core/server/ghost-server.js

```diff
@@ -46,7 +46,8 @@ GhostServer.prototype.closeConnections = function () {
 
 GhostServer.prototype.logStartMessages = function () {
     // Tell users if their node version is not supported, and exit
-    if (!semver.satisfies(process.versions.node, packageInfo.engines.node)) {
+    if (!semver.satisfies(process.versions.node, packageInfo.engines.node) &&
+        !semver.satisfies(process.versions.node, packageInfo.engines.iojs)) {
```

Comment:
This is somewhat nice and ugly.. For io.js versions we can use engines.iojs but there doesn't seem to be an easy way of determining the current runtime to only check for the io.js version iff were running io.js. Suggestions welcome!